### PR TITLE
Docs/update cer ref

### PR DESCRIPTION
**Pull request by zachferland**
Commits describe each set of changes, also addresses comments in https://github.com/ceramicnetwork/js-did/pull/121

### CODE REVIEW

File: docs/modules/dids.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ const { jws, linkedBlock } = await did.createDagJWS(payload)
 const jwsCid = await ipfs.dag.put(jws, { storeCodec: 'dag-jose', hashAlg: 'sha2-256' })
 
 // put the payload into the ipfs dag
-const block = await ipfs.block.put(linkedBlock, { cid: jws.link })
+const block = await ipfs.block.put(linkedBlock, { format: 'dag-cbor' })
```
Comment (oed) [inline docs/modules/dids.md:50]: Can you put a block with format? I thought blocks needed a CID?

**Review by oed — APPROVED** [review]
lgtm, thanks

File: docs/modules/dids.md (hunk 1 above)
Comment (zachferland) [reply docs/modules/dids.md:50]: not sure at what point it changed - https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/BLOCK.md#parameters-1

assuming it just uses codec/hash to derive cid it self instead now

Comment (oed) [reply docs/modules/dids.md:50]: ok 👍

**Pull request merged**